smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftWriter.kt

```diff
@@ -104,7 +105,7 @@ class SwiftWriter(
     fun addImport(symbol: Symbol) {
         symbol.references.forEach { addImport(it.symbol) }
         if (symbol.isBuiltIn || symbol.isServiceNestedNamespace || symbol.namespace.isEmpty()) return
```

Comment:
Below, `SwiftWriter` is modified to handle symbols that have multiple SPI names.